desc/plotting.py

```diff
@@ -3843,3 +3844,106 @@ def plot_logo(save_path=None, **kwargs):
         fig.savefig(save_path, facecolor=fig.get_facecolor(), edgecolor="none")
 
     return fig, ax
+
+
+def plot_gammac(eq, rhos=None, alphas=None, num_pitch=None):
+    """Plot the energetic proxy metric γ_c fo a single flux surface.
+
+    Parameters
+    ----------
+    eq : object
+        Equilibrium object containing magnetic field information
+    rhos : array_like or float, optional
+        Flux surface radii. If float, plots single surface.
+        Default: np.linspace(0.1, 1, 10)
+    alphas : array_like, optional
+        Fieldline label values (toroidal angle).
+        Default: np.linspace(0, 2π, 32, endpoint=True)
+    num_pitch : int, optional
+        Number of pitch angle values for bounce integral calculation.
+        Default: 16
+
+    Returns
+    -------
+    fig : matplotlib.figure.Figure
+        The figure object containing the plot
+    ax : matplotlib.axes.Axes
+        The axes object for further customization
+
+    """
+    if rhos is None:
+        rhos = np.array([0.5])
+    elif isinstance(rhos, float):
+        rhos = np.array([rhos])
+
+    print("Calculating γ_c(α, λ)..")
+    if alphas is None:
+        alphas = np.linspace(0, 2 * np.pi, 32, endpoint=True)
+
+    if num_pitch is None:
+        num_pitch = 16
+
+    # TODO(#1352)
+    X, Y = 32, 64
```

Comment:
might be useful to make these optional kwargs

Reply:
Done.